commit e39bd3cf62f02d2e1ab4ed717c34af732408e428 (refs/changes/79/4779/3)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2018-09-04T16:44:50-07:00 (1 year, 1 month ago)
    
    OS-7211 root should recognize pkgsrc in the gz
    Reviewed by: Jorge Schrauwen <sjorge@blackdot.be>
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Dan McDonald <danmcd@joyent.com>

diff --git a/overlay/generic/root/.profile b/overlay/generic/root/.profile
index 80e557e4..906d43ba 100644
--- a/overlay/generic/root/.profile
+++ b/overlay/generic/root/.profile
@@ -5,10 +5,10 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
-PATH=/usr/bin:/usr/sbin:/smartdc/bin:/opt/smartdc/bin:/opt/local/bin:/opt/local/sbin:/opt/smartdc/agents/bin
-MANPATH=/usr/share/man:/smartdc/man:/opt/smartdc/man:/opt/local/man
+PATH=/usr/bin:/usr/sbin:/smartdc/bin:/opt/smartdc/bin:/opt/local/bin:/opt/local/sbin:/opt/tools/bin:/opt/tools/sbin:/opt/smartdc/agents/bin
+MANPATH=/usr/share/man:/smartdc/man:/opt/smartdc/man:/opt/local/man:/opt/tools/man
 PAGER=less
 export PATH MANPATH PAGER
